tests/visualization_tests/matplotlib_tests/test_edf.py

```diff
@@ -11,6 +14,14 @@ def test_target_is_none_and_study_is_multi_obj() -> None:
         plot_edf(study)
 
 
+def _validate(y_values: List[float]) -> None:
```

Comment:
Maybe `Sequence` is preferrable if we can use it.
```suggestion
def _validate(y_values: Sequence[float]) -> None:
```

Reply:
Thank you for your advice. I will commit after modifying and checking.